### DISCUSSION TITLE
Search in article

### DISCUSSION THREAD

**Discussion by jborbik** [omnivore-app/omnivore#2823] [Q&A] [answered]
Hi all,
Is there a way now to search inside an article for the word occurrences? I am talking specifically about iOS version but web would be also relevant. 

Search seems to be available now only for all the saved items in the main menu but a more fine-grained search in an article would make the content navigation much easier.

Thanks!

**Comment by jacksonh** [comment, marked as answer]
@Jendker nothing that we provide however on iOS if you select a word you can choose "Find Selection", once you do that you can type anything you want to search for. This is just an iOS feature though, we should standardize it in our app instead.

**Reply by quinncomendant** [reply]
Here's screenshots of the “Find Selection” option. I just discovered it today, very secret. :)

![IMG_8855](https://github.com/user-attachments/assets/8f004ed6-f4cc-4b19-90e6-b4d72fc2799e)
![IMG_8854](https://github.com/user-attachments/assets/114d4683-857e-4b6e-b7c0-4534a027363d)

**Comment by quinncomendant** [comment]
I discovered the “Find Selection” option on iOS, which works perfectly. Now, we need the same feature in the macOS app! 🙏
